bin/pftt2-release.bat

```diff
@@ -0,0 +1,34 @@
+@ECHO OFF
+
+REM Check if build directory exists
+cd ../
+if exist build (
+	REM Create pftt-release directory in main folder
+	mkdir pftt-release
+
+	REM Copy contents of bin, conf and lib to respective folders
+	cd pftt-release
+	mkdir bin
+	mkdir conf
+	mkdir lib
+	cd ..
+
+	xcopy /s /i "./bin" "./pftt-release/bin"
+	xcopy /s /i "./conf" "./pftt-release/conf"
+	xcopy /s /i "./lib" "./pftt-release/lib"
+
+	REM Create pftt2.jar and place in lib of the package
+	cd build
+	jar cf pftt2.jar ./com/ ./org/columba ./org/kxml2 ./org/incava
```

Comment:
The pftt2.jar path can point directly to the package lib, spare the copy :)

Reply:
To clarify, the pftt2.jar should be made in the package lib directly so that it doesn't have to be moved from the build folder to the package lib?

Reply:
It works I mean, just a note on possible refinement. Like `%PFTT_LIB%\pftt2.jar` would eliminate need to copy. Using `-C %PFTT_HOME%\build` could eliminate need to change dir, etc.

Thanks

Reply:
I updated it to be one line now. It might look a little weird, but that's how I got it to work.

Reply:
I also updated most of the paths. Can you check if they're good now?